Dockerfile

```diff
@@ -18,6 +18,10 @@ RUN apt-get --no-install-recommends -y install \
     libmozjs185-dev \
     python
 
+# Install Nodejs
+RUN curl -sL https://deb.nodesource.com/setup | bash -	
```

Comment:
I feel cold for blindly executing arbitrary shell script. Isn't there a more secure and explicit way to add those repository?